--- default.settings.php	2010-10-11 18:49:48.000000000 -0500
+++ settings.php	2011-05-07 13:40:12.000000000 -0500
@@ -178,7 +178,18 @@
  *   );
  * @endcode
  */
-$databases = array();
+$databases = array(
+    'default' => array(
+        'default' => array (
+            'driver'   => 'mysql',
+            'database' => 'drupal',
+            'username' => 'drupal',
+            'password' => '',
+            'host'     => 'localhost',
+            'prefix'   => '',
+        )
+    )
+);
 
 /**
  * Access control for update.php script.
